app/utils/app_info.py

```diff
@@ -55,8 +56,17 @@ def __init__(self, main_file: Optional[str] = None) -> None:
         # Application metadata
 
         self._app_name = "RimSort"
-        self._app_version = ""
         self._app_copyright = ""
+        
+        self._app_version = "Unknown version"
+        version_file = str(self._application_folder / "version.xml")
```

Comment:
No changes needed: if these strings are used elsewhere, consider having these as constants or moving config values to a separate configuration file or module.